src/main/java/com/shaft/gui/element/WebDriverElementActions.java

```diff
@@ -1146,6 +1147,26 @@ public static void switchToWindow(WebDriver driver, String nameOrHandle) {
             WebDriverElementActions.failAction(driver, nameOrHandle, null);
         }
     }
+    /**
+     * Switches focus to another Tap
+     *
+     * @param driver       the current instance of Selenium webdriver
+     * @param URL The name of the URL you want to navigate to
+     */
+    public static void switchToNewTap(WebDriver driver, String URL) {
```

Comment:
the method should validate that the link has been opened in a new tab successfully.
this required getting window handles before and after opening the new tab, and confirming that the new tab was opened, printing the new tab window handle in the console using logdiscrete

Reply:
Done(Y)